CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="3.2.0"></a>
+
+## [3.2.0] - 2023-03-17
+
+### Changed
+
+- ListCalls allows for applicationId and has_application query parameters
```

Comment:
incorrect param reference (inconsistent). `has_application` should be `hasApplication`